src/main/scala/Matrix.scala

```diff
@@ -0,0 +1,8 @@
+import scala.util.Random
+
+object Matrix {
+  def matrix() : Unit = {
+    var alphanum = Random.alphanumeric
+    println(alphanum.nextInt())
```

Comment:
you don't need to have a variable to ask for a nextInt(), you could get rid of it and call ir directly, also it should be print instead of a println and it should be iterating, so in some loop